src/app/core/config-setup/config-import-parser.service.ts

```diff
@@ -0,0 +1,319 @@
+    return newColumnGroup;
+  }
+
+  private generateOrUpdateDetailsViewConfig(
+    fieldDef: ConfigFieldRaw,
+    entityType: string,
+    fieldId: string
+  ) {
+    if (
+      !fieldDef.show_in_details ||
+      fieldDef.show_in_details.toString().length === 0
+    ) {
+      return;
+    }
+
+    const detailsView: EntityDetailsConfig =
+      (this.generatedViews.get(entityType + "/:id")
+        ?.config as EntityDetailsConfig) ??
+      this.generateEmptyDetailsView(entityType + "/:id", entityType);
+
+    if (fieldDef.show_in_details) {
+      const panel: PanelComponent = this.generateOrFindDetailsPanel(
+        detailsView,
+        fieldDef.show_in_details.trim()
+      );
+      panel.config.cols[0].push(fieldId);
+    }
+  }
+
+  private generateEmptyDetailsView(
+    viewId: string,
+    entityType: string
+  ): EntityDetailsConfig {
+    const newDetailsView = {
+      entity: entityType,
+      icon: "child",
+      panels: [],
+      title: "",
+    };
+    this.generatedViews.set(viewId, {
+      _id: viewId,
+      component: "EntityDetails",
+      config: newDetailsView,
+    });
+    return newDetailsView;
+  }
+
+  private generateOrFindDetailsPanel(
+    detailsView: EntityDetailsConfig,
+    panelName: string
+  ): PanelComponent {
+    const existingPanel = detailsView.panels.find(
+      (c) => c.title === panelName && c.components[0].component === "Form"
+    );
+    if (existingPanel) {
+      return existingPanel.components[0];
+    }
+
+    const newPanel: Panel = {
+      title: panelName,
+      components: [{ title: "", component: "Form", config: { cols: [[]] } }],
+    };
+    detailsView.panels.push(newPanel);
+    return newPanel.components[0];
+  }
+
+  private generateDefaultConfigs(entityName: string) {
+    const configs = new Map<string, any>();
+
+    configs.set("appConfig", defaultJsonConfig.appConfig);
+    configs.set(
+      "appConfig:usage-analytics",
+      defaultJsonConfig["appConfig:usage-analytics"]
+    );
+    configs.set("navigationMenu", defaultJsonConfig.navigationMenu);
+    configs.set("view:", defaultJsonConfig["view:"]);
+    configs.set(
+      "enum:interaction-type",
+      defaultJsonConfig["enum:interaction-type"]
+    );
+    configs.set(
+      "enum:warning-levels",
+      defaultJsonConfig["enum:warning-levels"]
+    );
+    configs.set("view:note", defaultJsonConfig["view:note"]);
+    configs.set("view:admin", defaultJsonConfig["view:admin"]);
+    configs.set(
+      "view:admin/conflicts",
+      defaultJsonConfig["view:admin/conflicts"]
+    );
+    configs.set(
+      "view:admin/config-import",
+      defaultJsonConfig["view:admin/config-import"]
+    );
+    configs.set("view:import", defaultJsonConfig["view:import"]);
+    configs.set("view:user", defaultJsonConfig["view:user"]);
+    configs.set("view:user/:id", defaultJsonConfig["view:user/:id"]);
+    configs.set("view:help", defaultJsonConfig["view:help"]);
```

Comment:
Should we maybe add those views more centrally, not only through config, because some of them are (almost) hard-wired?

Reply:
View like `user`, `user/:id` and `help` still allow configuration. Therefore it still makes sense to have them in the config. For the `import` it might be relevant to assign permissions.